app/public/static/js/components/tutorial-instructions.js

```diff
@@ -0,0 +1,101 @@
+AFRAME.registerComponent("tutorial-instructions", {
+  multiple: true,
+
+  init: function () {
+    this.buttons = ["a", "w", "s", "d"];
+    this.dir = ["left", "forward", "backward", "right"];
+    const el = this.el;
+    this.cur_idx = 0;
+    this.keyDownHandle = this.keyDownHandle.bind(this);
+
+    // Create textLabel
+    this.textLabel = document.createElement("a-text");
+    this.textLabel.setAttribute("id", "text-instructions");
+    this.textLabel.setAttribute(
+      "value",
+      'Press "' +
+        this.buttons[this.cur_idx] +
+        '" to move ' +
+        this.dir[this.cur_idx]
+    );
+    this.textLabel.setAttribute(
+      "font",
+      "https://raw.githubusercontent.com/jaydhulia/aframe-fonts/master/fonts/poppins/Poppins-Medium.json"
+    );
+    this.textLabel.setAttribute("shader", "msdf");
+    this.textLabel.setAttribute("color", "white");
+    this.textLabel.setAttribute("align", "center");
+    this.textLabel.setAttribute("position", { x: 0, y: 0, z: -1 });
+    this.textLabel.setAttribute("width", 1);
+
+    el.appendChild(this.textLabel);
+    window.addEventListener("keydown", this.keyDownHandle);
+  },
+  keyDownHandle: function (e) {
+    const buttons = this.buttons;
+    const dir = this.dir;
+    if (e.key === buttons[this.cur_idx]) {
```

Comment:
can we also add a check for the CAPS WASD? If someone has caps lock on then they would not get past the first part i think

Reply:
Updated, by forcing e.key to be lower case.